packages/connectivity/src/scp-cf/destination/proxy-util.spec.ts

```diff
@@ -1,7 +1,7 @@
-import { unmockAllTestDestinations } from '@sap-cloud-sdk/test-util';
 import { createLogger } from '@sap-cloud-sdk/util';
 import { Protocol } from '../protocol';
 import { basicHeader } from '../authorization-header';
+import { unmockAllTestDestinations } from '../../../../test-util/src';
```

Comment:
I fail to recognize where the counterpart is called (`mockAllTestDestinations`). A quick search on my local state reveals that this is only called in the integration tests and in the tests of `test-util`. I wonder why we even need to call this...